Corona-Warn-App/src/main/res/layout/vaccination_list_incomplete_top_card.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    style="@style/Card.Vaccination"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginHorizontal="24dp"
+    android:layout_marginTop="16dp"
+    android:orientation="vertical"
+    android:paddingHorizontal="16dp"
+    android:paddingTop="24dp"
+    android:paddingBottom="16dp">
+
+    <TextView
+        android:id="@+id/top_card_title"
+        style="@style/headline4Bold"
```

Comment:
Text seems to large, figma shows 16sp, so this would be more like body1bold

Reply:
Adjusted the text sizes